build/Dockerfile

```diff
@@ -1,4 +1,21 @@
+ARG BUILD_ARCH=amd64
```

Comment:
please separate into 2 commits:
1. moving repo go build to container (+1 btw)
2. Adding multi arch thing..

Reply:
Done.

Reply:
You did so but the order of the commits is wrong. 
Please separate to 2 commits. The first commit only moves the repo go build to a container, setting the ground to the main commit. The second does the main thing this PR is meant to do which is adding the multi arch